docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+---
+version: '3.7'
+services:
+  node1:
+    container_name: node1
+    image: hazelcast/hazelcast:3.12.5
+
+  node2:
+    container_name: node2
+    image: hazelcast/hazelcast:3.12.5
+
+  hazelcast-quarkus1:
+    container_name: hazelcast-quarkus1
+    image: hazelcast-guides/hazelcast-quarkus-native
+    environment:
+      - CONTAINER_NAME=hazelcast-quarkus_1
+      - HAZELCAST_IP=node1
+    ports:
+      - "8080:8080"
+    depends_on:
+      - node1
+      - node2
+
+  hazelcast-quarkus2:
+    container_name: hazelcast-quarkus2
+    image: hazelcast-guides/hazelcast-quarkus-native
+    environment:
+      - CONTAINER_NAME=hazelcast-quarkus_2
+      - HAZELCAST_IP=node2
```

Comment:
let's just rename it to `hazelcast_member1,hazelcast_member_2`. I checked the doc and it seems comma seperated list of members should work fine with our config.